### PR TITLE
Added minutes and slides for 2024-01-25 TAC meeting

### PR DESCRIPTION
**Pull request by bensternthal**
This PR includes the meeting artifacts from the 2024-01-25 TAC meeting. @Riaankl

### CODE REVIEW

File: TAC/Meetings/2024/CCC-TAC-Minutes-2024-01-25.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+* Bokdeuk Jeong (Samsung)
+* Chris Ramming (Broadcom)
+* Drasko Draskovic (Ultraviolet)
+* Henk Birkholz (Fraunhofer Institute)
+* Kanth Ghatraju (Oracle) 
+* Marc Meunier (Arm)
+* Mark Novak (JP Morgan Chase)
+* Matthieu Legré (Cysec)
+* Ofir Azoulay-Rozanes (Anjuna)
+* Steveb Walli (Microsoft)
```
Comment (dcmiddle) [inline TAC/Meetings/2024/CCC-TAC-Minutes-2024-01-25.md:70]:
```suggestion
* Stephen Walli (Microsoft)
```

File: TAC/Meetings/2024/CCC-TAC-Minutes-2024-01-25.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+* Sung Lee (Broadcom)
+* Thomas Fossati (Linaro)
+* Wenhui Zhang (Bytedance/TikTok)
+* Mike Bursell (CCC)
+* Sal Kimmich (LF)
```
Comment (dcmiddle) [inline TAC/Meetings/2024/CCC-TAC-Minutes-2024-01-25.md:75]: Doesn't Sal have the same affiliation as Mike?

File: TAC/Meetings/2024/CCC-TAC-Minutes-2024-01-25.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+Mike Bursell (MB) reviewed comments and changes on NIST SP 800-226 and the content that he has rewritten. Discussion ensued. 
+
+## 2024 TAC Priorities
+
+DM presented the overall objective of the CCC TAC. Discussion ensued. DM then went on to review the 2024 objectives and key results. Additional notes were taken in the OKR Google doc.
+
+## New Project Onboarding
+
+DM reviewed the new project pipeline. DM went on to review the list of projects and which projects are missing mentors. Chris Ramming (CR) asked about how to best leverage CCC/LF resources including managing meetings. DM showed example documents. Action Item: It was agreed that there should be a project benefits section on the website.
```
Comment (dcmiddle) [inline TAC/Meetings/2024/CCC-TAC-Minutes-2024-01-25.md:110]: Did we suggest an owner for this action?

File: TAC/Meetings/2024/CCC-TAC-Minutes-2024-01-25.md (hunk 2 above)
Comment (bensternthal) [reply TAC/Meetings/2024/CCC-TAC-Minutes-2024-01-25.md]: Good catch.

File: TAC/Meetings/2024/CCC-TAC-Minutes-2024-01-25.md (hunk 3 above)
Comment (bensternthal) [reply TAC/Meetings/2024/CCC-TAC-Minutes-2024-01-25.md:110]: I did not catch one in the meeting recording. I would suggest Riaan.

**Pull request merged**